bundles/org.openhab.ui/web/src/assets/definitions/blockly/blocks-http.js

```diff
@@ -183,4 +266,23 @@ export default function (f7, isGraalJs) {
       throw new Error(unavailMsg)
     }
   }
+
+  function encodeParams (params) {
```

Comment:
I would prefer to have this code inside something like a HTTP request builder inside openhab-js, but it‘s fine for me to merge this PR as is and adjust the code later once we have implemented this.
I will have limited time the next weeks so I won‘t implement that anytime soon (and hopefully don‘t forget that).